src/css/footer.css

```
@@ -308,36 +304,33 @@ section#main-footer .section-container{
     padding: 0;
 }
 
-.footer-socials ul li a{
+.footer-socials ul li a {
```

```suggestion
.footer-socials a {
```

Dit is korter en doet hetzelfde.